src/SSHLibrary/pythonclient.py

```diff
@@ -80,6 +81,26 @@ def _login_with_public_key(self, username, key_file, password):
                                 allow_agent=False, timeout=float(self.config.timeout))
         except paramiko.AuthenticationException:
             raise SSHClientException
+        self.banner = self.client.get_transport().get_banner()
+
+    def get_banner(self):
+        return self.banner
```

Comment:
Is there a need to read banner right after login and set it into `self.banner`? If not, perhaps that attribute could be removed altogether and `get_banner()` could be implemented like `return self.client.get_transport().get_banner()`.

Could then also change `get_banner()` to a property `banner`, perhaps  it's better to keep it as a method because we have also `get_pre_login_banner()` method.